projects/ngx-formentry/src/form-entry/value-adapters/encounter.adapter.ts

```diff
@@ -150,6 +150,7 @@ export class EncounterAdapter implements ValueAdapter {
         payload,
         form.valueProcessingInfo.encounterTypeUuid
       );
+      this.setPayloadEncounterDateUuid(payload, form.valueProcessingInfo.encounterDatetime, form.valueProcessingInfo.utcOffset);
```

Comment:
This should be in it's own, separate guarded clause to ensure:

1. That the payload encounterDatetime isn't already set (e.g., by an `encounterDateTime` node).
2. That the `form.valueProcessingInfo.encounterDatetime` is populated with a value. Unless, are we relying on `null` returning today's date and time?

Reply:
Done